scripts/install_third_party.py

```diff
@@ -201,8 +201,8 @@ def test_manifest_syntax(dependency_type, dependency_dict):
 
     Display warning message when there is an error and terminate the program.
     Args:
-      dependency_type: dependency download format.
-      dependency_dict: manifest.json dependency dict.
+      dependency_type: dependency_download_format.
```

Comment:
dependency_download_format is not a type. Could you please go through all your changes here and make sure that the types and descriptions are correct?

Reply:
Should the type here be "*"?

Reply:
I think it should be `str` since `test_manifest_syntax` is called in `validate_manifest`. `validate_manifest` passes it `download_format` which is separated from `dependencies`. `dependencies` are separated from `manifest_data`, a JSON object.
@kevinlee12: Can you please see this once? Thanks!